config/imsim-config.yaml

```diff
@@ -133,7 +133,12 @@ image:
 
     det_name: "$det_name"
 
-    bandpass: { type: RubinBandpass, band: "$band" }
+    bandpass:
+        type: RubinBandpass
+        band: $band
+        # You can optionally add an airmass here to incorporate into the bandpass.  If you leave
+        # this out, then the fiducial bandpass with airmass = 1.2 will be used.
+        airmass: { type: OpsimData, field: airmass }
```

Comment:
Does using this config as-is mean that the default QE curve (as captured in `baseline/total_{band}.dat`) will be used?  If we want to use the per-CCD QE curves, what would the corresponding entries for `camera` and `det_name` be?  I guess it would be something like
```
camera: "$output.camera"
det_name: "$det_name"
```
?

Reply:
I think something like that will work (though maybe it's `@output.camera`?)  I'll double check.  Do you think we should make this (+ using the opsim airmass) the default?

Reply:
Yes, I think so.  We should always be specify the airmass.  I double checked that the instanceCat interface also calculates the airmass.  So, assuming it gets correctly into the opsim entry it should be good everywhere.

Reply:
Okay, I fiddled with some of the tests to make this work, but I think it's okay now.  (Apparently the throughput of the real LsstCam R22_S11 is better than what's in the throughputs repo).